.github/actions/nm-build-docker/action.yml

```diff
@@ -27,6 +27,11 @@ runs:
              --build-arg build_type=${{ inputs.build_type }} \
              --build-arg build_version=${{ inputs.build_version }} \
              --target vllm-openai . || status=$?
+      if [ ${status} -eq 0 ] && [[ "${build_type}" = "release" ]]; then
```

Comment:
nit: we've been using all caps elsewhere, e.g NIGHTLY or RELEASE or REMOTE.

Reply:
nice catch, updated